appeals/web/src/server/lib/nunjucks-filters/status-tag.js

```diff
@@ -10,7 +10,6 @@ export function appealStatusToStatusTag(appealStatus) {
 	return capitalizeFirstLetter(
 		appealStatus
 			.replace('issue_determination', 'issue_decision')
-			.replace('lpa_questionnaire_due', 'lpa_questionnaire')
```

Comment:
Probably also keep this for now? See below...

Reply:
Done - I've also updated `mapAppealStatusToActionRequiredHtml` to check for the old value as well as the new one, so should handle both until the status can be globally updated